integration/test-integration/src/test/resources/openAPIs/prod_and_sand_openAPI.yaml

```diff
@@ -66,7 +66,7 @@ paths:
         - http://mockBackend:2390/
     x-wso2-sandbox-endpoints:
       urls:
-        - http://mockBackend:2383
+        - http://mockBackend:2383/
```

Comment:
Why do you need / here?

Reply:
We are striping the trailing slash when we process the endpoints in the adapter. https://github.com/wso2/product-microgateway/blob/393255b3ca37d51b6f89a502100740a51b586009/adapter/internal/oasparser/envoyconf/routes_with_clusters.go#L95

So I think it doesn't make any difference whether we add the slash there or not. I just added it there to make it consistent with the corresponding production endpoint (it also contains a trailing slash, line 66). I tested it manually and also did the CI with and without trailing slash to check whether there are any test cases failing. Should I remove the trailing slash here ? Also in the corresponding production endpoint ?